Fix mathjax in formgrade templates

brichet opened the pull request:
This PR fix an error when loading MathJax in formgrade template.
*HTMLExporter* in `nbconvert>6` replace the variable `resources['mathjax_url']` with https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/latest.js?config=TeX-AMS_CHTML-full,Safe.
That was breaking the loading of *MathJax* in formgrade template.

jhamrick commented:
Thanks for tracking this down!

jhamrick commented:
Can you update this PR to merge into master, rather than 0.7.x? That way we get the fix for 0.8 too (I will backport it to 0.7.x). I changed the base but it introduced merge conflicts as the PR includes other stuff from the 0.7.x branch that isn't on main.

brichet commented:
> Can you update this PR to merge into master, rather than 0.7.x? That way we get the fix for 0.8 too (I will backport it to 0.7.x). I changed the base but it introduced merge conflicts as the PR includes other stuff from the 0.7.x branch that isn't on main.

@jhamrick I rebased the PR on the main branch.

jhamrick commented:
@meeseeksdev backport to 0.7.x
